### ISSUE TITLE
Feature: support historic information via services.opengeo.nl

### ISSUE THREAD

**Issue by skinkie** [plannerstack/PTDS.js#33]
I am aiming for a selection of a minimum _date picker_, and potentially selecting an available network. For now three files per day are provided.

**Comment by joined** [comment]
It would be nice if https://services.opengeo.nl/ptds/ had a JSON file describing the available datasets, otherwise I have to do HTML parsing in Javascript (not fun) if I want to make the date/network picker dynamic

Something like:

```json
[
  {
    "date": "2018-03-01",
    "url": "https://services.opengeo.nl/ptds/2018-03-01/",
    "datasets": [
      {
        "filename": "1-9-11-12-16-17.json",
        "lines": [1, 9, 11, 12, 16, 17]
      },
      {
        "filename": "1-15-19.json",
        "lines": [1, 15, 19]
      },
      {
        "filename": "2-3-4-6-44-83.json",
        "lines": [2, 3, 4, 6, 44, 83]
      }
    ]
  },
  {
    "date": "2018-03-02",
    "url": "https://services.opengeo.nl/ptds/2018-03-02/",
    "datasets": [
      {
        "filename": "1-9-11-12-16-17.json",
        "lines": [1, 9, 11, 12, 16, 17]
      },
      {
        "filename": "1-15-19.json",
        "lines": [1, 15, 19]
      },
      {
        "filename": "2-3-4-6-44-83.json",
        "lines": [2, 3, 4, 6, 44, 83]
      }
    ]
  }
]
```

**Comment by joined** [comment]
Ah well of course I can also hardcode the datasets in the codebase, but it would mean having to update it every time you change something in your service

**Comment by skinkie** [comment]
I'll create a file as such. Might also be a start for a network description.

**Comment by skinkie** [comment]
https://services.opengeo.nl/ptds/index.json

Would an index by date maybe a better idea?

**Comment by joined** [comment]
Doesn't make that big of a difference, logically I think an array makes more sense because it encodes the order, and the datasets are instrinsically ordered by date. Objects in javascript do not guarantee the order of the properties, so I would have to take care of that myself

**Comment by joined** [comment]
I would say the most important part of this feature was implemented in the now-merged #38.
If we want more advanced parameters for the visualization, we also need a new structure for the index file as discussed

**Comment by skinkie** [comment]
I consider this implemented.
